src/Dropdown.js

```diff
@@ -6,11 +6,12 @@ import PropTypes from 'prop-types';
 import ReactDOM from 'react-dom';
 import { Manager } from 'react-popper';
 import classNames from 'classnames';
-import { mapToCssModules, omit, keyCodes } from './utils';
+import { mapToCssModules, omit, keyCodes, deprecated } from './utils';
 
 const propTypes = {
   disabled: PropTypes.bool,
-  dropup: PropTypes.bool,
+  dropup: deprecated(PropTypes.bool, 'Please use the prop "direction"'),
```

Comment:
I would also indicate the using the value "up": `'Please use the prop "direction" with the value "up".'`